x-pack/plugins/beats_management/public/config_schemas.ts

```diff
@@ -310,6 +310,86 @@ const metricbeatModuleConfig: YamlConfigSchema[] = [
   },
 ];
 
+const heartbeatConfig: YamlConfigSchema[] = [
+  {
+    id: 'type',
+    ui: {
+      label: 'Type',
+      type: 'select',
+    },
+    options: [
+      {
+        value: 'http',
+        text: 'http',
+      },
+      {
+        value: 'icmp',
+        text: 'icmp',
+      },
+      {
+        value: 'tcp',
+        text: 'tcp',
+      },
+    ],
+    error: 'Please select a monitor type',
```

Comment:
Please use a translation token for this

Reply:
When I first committed these changes we weren't using translation tokens, will update ++